### ISSUE TITLE
scikit learn version mismatch between node and researcher raises error

### ISSUE THREAD

**Issue by srcansiz** [fedbiomed/fedbiomed#293]
In GitLab by @sharkovsky on May 17, 2022, 16:01

Some time ago I installed some packages in my researcher environment. This triggered an update of scikit-learn to version 1.0.2 on the researcher side, while the version on the node remained 0.24.2

Ultimately, this resulted in a different default value for the name of the `loss` parameter, causing the training to fail while instantiating the SGDRegressor (see the name change in the warning box of the [documentation](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDRegressor.html) -- `squared_loss`was deprecated in v1.0 )

I see that in our conda environments we enforce version 0.24.2 on both researcher and node. We should discuss if we want to take any action or if we simply assume that the researcher is not allowed to modify their conda environment.
